base/exports.jl

```diff
@@ -1063,3 +1063,67 @@ export
     @view,
     @views,
     @static
+
+# TODO: use normal syntax once JuliaSyntax.jl becomes available at this point in bootstrapping
+eval(Expr(:public,
+    Symbol("@assume_effects"),
+    Symbol("@constprop"),
+    Symbol("@locals"),
+    Symbol("@propagate_inbounds"),
+    :AbstractLock,
+    :AsyncCondition,
+    :CFunction,
+    :Checked,
+    :CodeUnits,
+    :DL_LOAD_PATH,
+    :EnvDict,
+    :Event,
+    :Fix1,
+    :Fix2,
+    :Generator,
+    :ImmutableDict,
+    :IteratorEltype,
+    :IteratorSize,
+    :OneTo,
+    :Pairs,
+    :Semaphore,
+    :UUID,
+    :acquire,
+    :active_project,
+    :add_sum,
+    :cconvert,
+    :checked_length,
+    :elsize,
+    :exit_on_sigint,
+    :front,
+    :identify_package,
+    :include,
```

Comment:
```suggestion
```
(this is probably private, in that it even partially deletes itself after finishing loading Base)